extensions/interactions/ImageClickInput/directives/image-click-input-interaction.directive.html

```diff
@@ -40,29 +40,29 @@
        ng-style="$ctrl.imageContainerStyle">
   <div ng-if="$ctrl.highlightRegionsOnHover">
     <div ng-repeat="region in $ctrl.allRegions"
-         style="
-           height: <[$ctrl.getRegionDimensions($index).height]>px;
-           width: <[$ctrl.getRegionDimensions($index).width]>px;
-           left: <[$ctrl.getRegionDimensions($index).left]>px;
-           top: <[$ctrl.getRegionDimensions($index).top]>px;
-           pointer-events: none;
-           position: absolute;
-           display: <[$ctrl.getRegionDisplay(region.label)]>;
-           background: rgba(0,0,255,0.5);
-           z-index: 10;">
+         ng-style="{
+           'height': $ctrl.getRegionDimensions($index).height + 'px',
+           'width': $ctrl.getRegionDimensions($index).width + 'px',
+           'left': $ctrl.getRegionDimensions($index).left + 'px',
+           'top': $ctrl.getRegionDimensions($index).top + 'px',
+           'pointer-events': 'none',
```

Comment:
One note (thanks to @DubeySandeep for spotting this!) -- could you please put the non-interpolated stuff in a class, and use ng-style only for the ones that need it? E.g. here, pointer-events, position, background, z-index can go in a class; the rest can stay in ng-style.

Ditto below.